phalcon/Cli/Console.zep

```diff
@@ -34,13 +34,17 @@ class Console extends AbstractApplication
     /**
      * Handle the whole command-line tasks
      */
-    public function handle(array arguments = null)
+    public function handle(var! arguments)
     {
         var className, container, dispatcher, eventsManager, module, moduleName,
             moduleObject, modules, path, router, task;
 
         let container = this->container;
 
+        if unlikely typeof arguments != "array" {
+            throw new Exception("Argument should be an array");
```

Comment:
`public function handle(array arguments = null)` solves the task

Reply:
And do what Serghei says - remove the exception checking also